src/app/(main)/EventPage.tsx

```diff
@@ -115,7 +115,7 @@ export const EventPage = ({ eventId }: { eventId: string }) => {
             <OutputTime label="End Time" time={activity.endTime}></OutputTime>
           </Box>
         </OutputForm>
-        <OutputTextArea label="Description" value={activity.description}></OutputTextArea>
+        <OutputTextArea label="Description" value={activity.description} maxHeight={3}></OutputTextArea>

```

Comment:
Maybe instead of maxHeight something like linesVisible would be more descriptive? MaxHeight immediately makes me think of pixels or em units, not lines.

Reply:
I'm going to use `rows` to align with the `textarea` prop name.